distribution/helm/vector-agent/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+{{ .Chart.Name }} has been installed. Check it's status by running:
+  $ kubectl --namespace {{ .Release.Namespace}} get pods -l "app.kubernetes.io/instance={{ .Release.Name }}"
+
+WARNING: The `rawConfig` key has been deprecated and will be removed in an upcoming release.
```

Comment:
Maybe we could add a link to the highlight that will be published? The naming is predictable, for example: `https://vector.dev/highlights/2021-04-21-vector-tap/`